server/py/framework/db/sqldb/helpers.py

```diff
@@ -87,6 +89,19 @@ def generate_query_predicate_for_name(column, query_string):
         return column.__eq__(query_string)
 
 
+def generate_query_for_name_with_wildcard(column, query_string):
+    if query_string.startswith("~"):
+        return column.ilike(translate_wildcard_to_sql(query_string[1:]))
+    else:
+        column.__eq__(query_string)
```

Comment:
it doesn't return anything when the query_string does not start with ~

Reply:
@moranbental nice catch!